modules/elt/pages/use-airbyte-to-load-data-from-external-sources-to-teradata-vantage.adoc

```diff
@@ -0,0 +1,209 @@
+
+[NOTE]
+====
+For more details, please refer link:https://docs.airbyte.com/integrations/sources/google-sheets/#:~:text=For%20Airbyte%20Open%20Source%3A[Setting Google Sheets as Source Connector in Airbyte Open Source, window="_blank"] 
+====
+
+* Click Set up source, if the configuration is correct, you will get the message `All connection tests passed!`
+
+
+=== Setting the Destination Connection
+* Assuming you want to create a fresh new connection with `Teradata Vantage`, Select `Teradata Vantage` as the destination type under the "Set up the destination" section.
+* Add the `Host`, `User`, and `Password`. These are the same as the `Host`, `Username`, and `Password` respectively, used by your Clearscape Analytics Environment.
+* The `Default Schema` name is optional, however, you can provide a more appropriate name to your specific context. Here we have provided `gsheet_airbyte_td`.
+
+[NOTE]
+====
+If you do not provide a `Default Schema`, you will get an error stating "Connector failed while creating schema". Make sure you provide appropriate name in the `Default Schema`.
+====
+
+
+image::{dir}/configuring_destination_teradata_airbyte.png[Configuring the destination Teradata in Airbyte]
+
+
+* Click Set up destination, if the configuration is correct, you will get the message `All connection tests passed!`
+
+
+[NOTE]
+====
+You might get a configuration check failed error. Make sure your Teradata Vantage instance is running properly before making a connection through Airbyte. 
+====
+
+=== Configuring Data Sync
+Namespace is a group of streams (tables) in a source or destination. A schema in a relational database system is an example of a namespace. In a source, the namespace is the location from where the data is replicated to the destination. In a destination, the namespace is the location where the replicated data is stored in the destination.
+For more details please refer link:https://docs.airbyte.com/understanding-airbyte/namespaces/[Airbyte Namespace., window="_blank"]
+
+
+image::{dir}/namespaces_in_destination.png[Namespaces in the destination]
+
+
+In our example the destination is a database, so the namespace is the default schema `gsheet_airbyte_td` we defined when we configured the destination. The stream name is a table that is mirroring the name of the spreadsheet in the source, which is `sample_employee_payrate` in this case. Since we are using the single spreadsheet connector, it only supports one stream (the active spreadsheet). 
+
+Other type of sources and destinations might have a different layout. In this example, Google sheets as source does not support a namespace. 
+In our example, we have used `<destination schema>` as the Namespace of the destination, this is the default namespace assigned by Airbyte based on the `Default Schema` we declared in the destination settings. The database `gsheet_airbyte_td` will be created in our Teradata Vantage Instance.
+
```

Comment:
[NOTE]
====
We use the term "schema", as it is the term used by Airbyte. In a Teradata context the term "database" is the equivalent.
====